src/panels/lovelace/components/hui-image.ts

```diff
@@ -161,6 +161,14 @@ export class HuiImage extends LitElement {
       imageSrc = this.image;
     }
 
+    if (
+      this._loadState === LoadState.Loading &&
+      !this.cameraImage &&
+      imageSrc
+    ) {
+      this._loadState = LoadState.Loaded;
+    }
```

Comment:
Render should not have side effects, this should be done in `willUpdate`

Reply:
Good point!